tests/test_client_functional.py

```diff
@@ -876,8 +876,8 @@ async def redirect(request):
         raise web.HTTPFound(location="/")
 
     app = web.Application()
-    app.router.add_get("/", handler)
-    app.router.add_get("/redirect", redirect)
+    app.router.add_get("/", handler, allow_head=False)
+    app.router.add_get("/redirect", redirect, allow_head=False)
```

Comment:
Is this now required? That potentially makes it a backwards-incompatible change. Although, it looks weird that it was previously using `add_head()` separately anyway...

Reply:
Now required. But in any case next add_head creates an unreachable resource